src/app/sentry.ts

```diff
@@ -17,4 +17,9 @@
 // along with Runbox 7. If not, see <https://www.gnu.org/licenses/>.
 // ---------- END RUNBOX LICENSE ----------
 
-// ADD SENTRY HERE
+import { environment } from '../environments/environment'
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({
+  dsn: environment.SENTRY_DSN
```

Comment:
Could add to capture console.error messages:

      integrations: [Sentry.captureConsoleIntegration()],